demo/bike/main.js

```diff
@@ -1,13 +1,17 @@
-const {KalmanFilter} = kalmanFilter;// eslint-disable-line no-undef
-const createElement = require('../shared/views/create-element');
+/**
+ * @typedef {typeof import('../../index.d.ts')} kalmanFilter
+ */
+const { KalmanFilter } = /** @type {kalmanFilter} */ (kalmanFilter);// eslint-disable-line no-undef
```

Comment:
now `kalmanFilter` is properly typed with all the lib typing withing this JS file.

Reply:
ok